README.md

```diff
@@ -1,3 +1,5 @@
+[![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/git/git) 
```

Comment:
I only read the name `gitpod` in another git/git PR, and neither that PR nor this here PR manage to clarify to me what this is all about.

Please try your best to explain (in the commit message) to a person who has _never_ heard of gitpod why it is something Git should have, what problems it solves for the Git contributor community, and accompany it with a link to gitpod.